corehq/toggles.py

```diff
@@ -1758,3 +1758,11 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_CUSTOM,
     [NAMESPACE_USER]
 )
+
+
+MANAGE_CCZ_HOSTING = StaticToggle(
+    'manage_ccz_hosting',
+    'Allow project to configure ccz hosting',
+    TAG_CUSTOM,
+    [NAMESPACE_USER, NAMESPACE_DOMAIN]
+)
```

Comment:
Since this has `NAMESPACE_USER` always_enabled would not be correct right?

In terms of environment i think it would be needed on india and icds and may be staging. I think i am fine to keep it open to other environments.

Reply:
The "enabled" checks look for a user OR domain match.  If it's going to be enabled for a domain, then `always_enabled` should work fine for that, and wouldn't preclude users from using it on other domains.

Reply:
Does this need to be both user and domain? I thought we were moving away from user-oriented flags?

Reply:
@esoergel 
> If it's going to be enabled for a domain

Ya thats why i am more keen on keeping it per user because if it's set for a domain, all users would start seeing the option to manage it and we wouldn't want that.

Reply:
@orangejenny 
> I thought we were moving away from user-oriented flags

Is it? User specific permissions should be handled via role/permissions?

Reply:
Domain-oriented flags are generally better because they tell us more about where a feature is in use. But if this needs to be a user-specific feature, feature flag is fine, I don't think you need to muck around in user permissions.

Reply:
Definitely agree that this shouldn't touch user permissions.  Can you drop the `NAMESPACE_DOMAIN`, then?  It sounds like turning this on for a whole domain would be a bad idea, especially since the views are open to any user on the domain as discussed [here](https://github.com/dimagi/commcare-hq/pull/24129/files#r281403565)

Reply:
@esoergel updated to keep just as user
https://github.com/dimagi/commcare-hq/pull/24129/commits/f7a63c0219825eab0c5c4e9f2cd2aa7b51ba5269